src/applications/income-limits/api/index.js

```diff
@@ -0,0 +1,24 @@
+const SERVER_ERROR_REGEX = /^5\d{2}$/;
+const CLIENT_ERROR_REGEX = /^4\d{2}$/;
+
+export const getData = async ({ zipCode, year, dependents }) => {
+  const DOMAIN = document.location.origin;
```

Comment:
This may be a problem with staging due to the fact that the API url uses a `-` instead of a `.` (https://staging-api.va.gov/income_limits/v1/limitsByZipCode/).  This will also be why we wont get a working result in RI I think.

The solution might be to set up an api config object similar to here `src/applications/facility-locator/config.js` that leverages the platform utilities constants to point at the right API paths